### PR TITLE
BoT/EoT chrono adaption

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**
* We now don't run into `chrono` panics when using TimeInterval::default()
* I had to fix the time snap when it ran out of bounds
* The raster-vector-join now emits nulls when querying with points that have TimeInterval::default(). It is quite slow because the GdalSource seems to collect every time interval from BoT to EoT. The aggregator, however, returns early if everything is null.
* I changed `From<i64>` to `TryFrom<i64>` for `TimeInstance`. This is more correct but somewhat unergonomic.

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Begin of time und EoT auf Chrono anpassen](https://trello.com/c/J4Kd9IlK/52-begin-of-time-und-eot-auf-chrono-anpassen)

File: datatypes/src/primitives/time_interval.rs [hunk 1]
```diff
@@ -35,8 +37,8 @@ impl Default for TimeInterval {
     /// ```
     fn default() -> Self {
         Self {
-            start: i64::min_value().into(),
-            end: i64::max_value().into(),
+            start: chrono::MIN_DATETIME.into(),
```
Comment (jdroenner) [inline datatypes/src/primitives/time_interval.rs:40]: can we use the min max from TimeInstancE?

**Comment by jdroenner** [comment]
i think it is fine. please unify the source of  max min values.

File: datatypes/src/primitives/time_interval.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply datatypes/src/primitives/time_interval.rs]: 5a4d480

**Comment by ChristianBeilschmidt** [comment]
0fc1dff changes the Vec to Iterator of tile parts such that the CI runs through.

**Pull request merged**